client/components/TestQueueRow/index.jsx

```diff
@@ -120,6 +127,24 @@ const TestQueueRow = ({
                 Published {gitUpdatedDateToString(testPlanVersion.updatedAt)}
             </p>
         );
+
+        const latestTestPlanVersion = latestTestPlanVersions.filter(
+            version => version.latestTestPlanVersion.id === testPlanVersion.id
+        );
+        const updateTestPlanVersionButton =
+            isAdmin && latestTestPlanVersion.length === 0 ? (
+                <Button
+                    className="updater-button"
+                    onClick={() => setShowTestPlanUpdaterModal(true)}
+                    size="sm"
+                    variant="outline-secondary"
+                >
+                    {' '}
+                    Update Test Plan Version
```

Comment:
Since it's the "Test Plan Updater" now I feel like this button can be called "Update Test Plan" which would also make it more compact. I like the new styling. Good job picking its location, so far everyone's been happy with it.

Also could you remove the `{' '}`?